CheckDrive.Web/CheckDrive.Web/Views/Cars/Index.cshtml

```diff
@@ -30,9 +35,9 @@
                         <a class="btn btn-info" href="@Url.Action("Index", "oilMarks")">Yoqilg`ilar</a>
                     </div>
                     <div>
-                        <a class="btn btn-success" asp-action="Create">
-                            <i class="fa fa-plus"></i> Yaratish
-                        </a>
+                        <button id="openCarCreateModal" type="button" data-toggle="modal" class="btn btn-success">
+                            <i  class="fa fa-plus"></i> Yaratish
+                        </button>                                   
```

Comment:
Was this button worked ? in my opinion this button does not work because u dont use type="submit" to submit your form

Reply:
this button is to release the pop-up

Reply:
OK